mock/py/mockbuild/podman.py

```diff
@@ -145,7 +143,7 @@ def get_layers_digest(self):
         # data which should be sufficient to confirm the image
         data = {
             'RootFS': data['RootFS'],
-            'Config': data['Config'],
+            'Config': data.get('Config'),
```

Comment:
Hmm, `podman image inspect fedora:41 --format "{{ .RootFS.Layers }} {{ .Config }}"` works.  Same for `quay.io/centos/centos:stream9`.

I think we need to access the first item in the upper-level array, like:
`podman image inspect quay.io/centos/centos:stream9 | jq '.[0].RootFS'`

Reply:
updated (and tested with fedora rawhide)

Reply:
works: https://artifacts.dev.testing-farm.io/2ce4be47-b383-4139-a622-29dbe950709f/